### PR TITLE
Finish Alphabetizing the other classes

### PR DESCRIPTION
**Pull request by jtsshieh**

### CODE REVIEW

**Comment by jtsshieh** [comment]
If all private methods need to be prefixed with an undescores, private methods in `VoiceConnection` and `SharedStream` have not been prefixed (I think I can assume they are private because they don't have jsdoc comments)

**Comment by jtsshieh** [comment]
Here's the current order that's enforced on most classes:

* Constructor
* Alphabetized getters 
* Alphabetized public methods
* Alphabetized private methods that start with an undescore
* toString
* toJSON

**Comment by jtsshieh** [comment]
Should that be enforced, or is there a better order

**Comment by jtsshieh** [comment]
Could use this to enforce it, but I'm not sure if we should add another dev dependency.
https://github.com/bryanrsmith/eslint-plugin-sort-class-members

File: lib/voice/SharedStream.js [hunk 1]
```diff
@@ -126,43 +133,12 @@ class SharedStream extends EventEmitter {
...
-                connection._sendPacket(connection._createPacket(this.current.buffer));
-            }
-        });
-        this.current.playTime += this.current.options.frameDuration;
-        this.current.timeout = setTimeout(this._send, this.current.startTime + this.current.pausedTime + this.current.playTime - Date.now());
+    /**
+    * Remove a voice connection from the shared stream
+    * @arg {VoiceConnection} connection The voice connection to remove
+    */
+    remove(_connection) {
```
Comment (bsian03) [inline lib/voice/SharedStream.js]: merge conflict

File: lib/voice/SharedStream.js [hunk 2]
```diff
@@ -15,6 +15,21 @@ try {
 /**
 * Represents a collection of VoiceConnections sharing an input stream
 * @extends EventEmitter
+* @prop {Object?} current The current stream
```
Comment (bsian03) [inline lib/voice/SharedStream.js:18]:
```suggestion
* @prop {Object} current The current stream
```

Comment (bsian03) [reply lib/voice/SharedStream.js:18]: nvm

File: lib/voice/SharedStream.js [hunk 3]
```diff
@@ -15,6 +15,24 @@ try {
...
+* @prop {Number?} current.options.frameDuration The resource opus frame duration (required for DCA/Ogg)
+* @prop {Number?} current.options.frameSize The resource opus frame size
+* @prop {Boolean?} current.options.inlineVolume Whether to enable on-the-fly volume changing. Note that enabling this leads to increased CPU usage
+* @prop {Array<String>?} current.options.inputArgs Additional input parameters to pass to ffmpeg/avconv (before -i)
+* @prop {Number?} current.options.sampleRate The resource audio sampling rate
+* @prop {Number?} current.options.voiceDataTimeout Timeout when waiting for voice data (-1 for no timeout)
+* @prop {Number} current.pausedTime How long the current stream has been paused for, in milliseconds
+* @prop {Number} current.pausedTimestamp The timestamp of the most recent pause
+* @prop {Number} current.playTime How long the current stream has been playing for, in milliseconds
+* @prop {Number} current.startTime The timestamp of the start of the current stream
```
Comment (bsian03) [inline lib/voice/SharedStream.js]:
```suggestion
```
We don't document properties of objects in class props

File: lib/util/Bucket.js [hunk 4]
```diff
@@ -77,6 +63,20 @@ class Bucket {
             }, this.tokens < this.tokenLimit ? this.latencyRef.latency : Math.max(0, this.lastReset + this.interval + this.tokenLimit * this.latencyRef.latency - Date.now()));
         }
     }
+
+    /**
+    * Queue something in the Bucket
+    * @arg {Function} func A callback to call when a token can be consumed
+    * @arg {Boolean} [priority] Whether or not the callback should use reserved tokens
```
Comment (bsian03) [inline lib/util/Bucket.js]:
```suggestion
    * @arg {Boolean} [priority=false] Whether or not the callback should use reserved tokens
```

**Pull request merged**